app/services/nytimes_service.rb

```diff
@@ -0,0 +1,19 @@
+require 'faraday'
+require 'json'
+
+class NytimesService
+  def get_obituaries
+    JSON.parse(conn.get.body, symbolize_names: true)
+  end
+
+  private
+
+  def conn
+    Faraday.new(url:'https://api.nytimes.com/svc/search/v2//articlesearch.json') do |f|
+      f.params['api-key'] = ENV['NYT_API_KEY']
+      f.params['fq'] = 'news_desk:("obituaries")'
+      f.params['fq'] = 'section_name:("obituaries")'
+    end
```

Comment:
This is where we could start trying different API requests

Reply:
Excellent

Reply:
As we discussed on the "telephone", section_name is probably our best bet at this point because news_desk results are much more limited, and section_name appears to include news_desk but a whole lot more.